src/main/java/com/fasterxml/jackson/databind/SerializerProvider.java

```diff
@@ -946,20 +946,17 @@ public JsonSerializer<Object> getDefaultNullValueSerializer() {
         return _nullValueSerializer;
     }
 
-    /**
-     * Method called to get the serializer to use for serializing
-     * Map keys that are nulls: this is needed since JSON does not allow
-     * any non-String value as key, including null.
-     *<p>
-     * Typically, returned serializer
-     * will either throw an exception, or use an empty String; but
-     * other behaviors are possible.
-     */
     /**
      * Method called to find a serializer to use for null values for given
      * declared type. Note that type is completely based on declared type,
      * since nulls in Java have no type and thus runtime type cannot be
      * determined.
+     * <p>
+     * This method serializes Map keys that are nulls.
```

Comment:
Not quite, it finds serializer used to serializer Map keys that are nulls.
Probably need to combine this paragraph with first one for it to make full sense.